Source/WebKit/UIProcess/API/Cocoa/_WKWebAuthenticationPanel.mm

```diff
@@ -384,16 +411,30 @@ + (void)deleteLocalAuthenticatorCredentialWithID:(NSData *)credentialID
 + (void)deleteLocalAuthenticatorCredentialWithGroupAndID:(NSString *)group credential:(NSData *)credentialID
 {
 #if ENABLE(WEB_AUTHN)
+    BOOL useAlternateKeychainAttribute = NO;
+#if USE(APPLE_INTERNAL_SDK)
+    useAlternateKeychainAttribute = shouldUseAlternateKeychainAttribute();
+#endif
     auto deleteQuery = adoptNS([[NSMutableDictionary alloc] init]);
     [deleteQuery setDictionary:@{
         (__bridge id)kSecClass: bridge_id_cast(kSecClassKey),
-        (__bridge id)kSecAttrApplicationLabel: credentialID,
         (__bridge id)kSecUseDataProtectionKeychain: @YES,
         (__bridge id)kSecAttrSynchronizable: (id)kSecAttrSynchronizableAny
     }];
     updateQueryForGroupIfNecessary(deleteQuery.get(), group);
 
+    if (useAlternateKeychainAttribute)
+        [deleteQuery setObject:credentialID forKey:(__bridge id)kSecAttrAlias];
+    else
+        [deleteQuery setObject:credentialID forKey:(__bridge id)kSecAttrApplicationLabel];
```

Comment:
You should assign `kSecAttrAlias` or `kSecAttrApplicationLabel` to a temporary based on a conditional, and avoid all of this bulky code duplication.

Reply:
Done